ca/config.go

```diff
@@ -135,6 +135,13 @@ func (s *SecurityConfig) UpdateRootCA(cert, key []byte, certExpiry time.Duration
 	return err
 }
 
+// loadNewRootCA replaces the root CA with a new root CA
+func (s *SecurityConfig) loadNewRootCA(rootCA *RootCA) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+	s.rootCA = rootCA
+}
```

Comment:
I was originally thinking about that, but I wondering whether the saving should explicitly happen in `RequestAndSaveNewCertificates` since that's also where the TLS certs are saved.  But I guess if the whole thing isn't necessarily atomic anyway (should it be?), that would be a fine place to move it.